adapter/internal/oasparser/utills/mgw_swagger_utils.go

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+// Package utills holds the implementation for common utility functions
+package utills
+
+import (
+	"strings"
+)
+
+// GetIsOrganizationInList returns whether the organization is in the list of organizations that has the circuit breaker.
+func GetIsOrganizationInList(orgID string, cbOrganizationList string) bool {
+
+	cbOrganizationsList := strings.Split(cbOrganizationList, ",")
+
+	// Convert the list to a set for O(1) lookup
+	cbOrganizationsSet := make(map[string]struct{}, len(cbOrganizationsList))
```

Comment:
Convert list to a set is O(n) still

Reply:
Still even O(n) is fine here I guess, since it is related to deployment time and we are talking about very few entries here.